includes/core-changes/windowsforms/5.0/invalid-args-cause-argumentexception.md

```diff
@@ -0,0 +1,40 @@
+### WinForms methods now throw ArgumentException
+
+Some Windows Forms methods now throw an <xref:System.ArgumentException> for invalid arguments, where previously they did not.
+
+#### Change description
+
+Previously, passing arguments of an unexpected or incorrect type to certain Windows Forms methods would result in an indeterminate state. Starting in .NET 5.0, these methods now throw an <xref:System.ArgumentException> when passed invalid arguments.
+
+Throwing an <xref:System.ArgumentException> conforms to the behavior of the .NET runtime. It also improves the debugging experience by clearly communicating which argument is invalid.
+
+The following table lists the affected methods and parameters:
+
+| Method | Parameter name | Condition | Version added |
+|-|-|-|
+| <xref:System.Windows.Forms.TabControl.GetToolTipText(System.Object)?displayProperty=fullName> | `item` | Argument is not of type <xref:System.Windows.Forms.TabPage>. | 5.0 Preview 1 |
```

Comment:
There's only one item in the table, but all the plurals in the preceding text lead you to expect more than that. Do you expect to add more later?

Reply:
Yes, there's one for preview 5 that I know about but don't want to document yet. Do you think it's okay as-is?

Reply:
In that case, yes.

Reply:
I think we can expect more to come as we go through the codebase